packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -507,14 +492,11 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     this.setState({ isCircleClicked: true });
   };
 
-  private _getOpacity = (selectedArea: string): number => {
```

Comment:
Is _getOpacity applicable for legends

Reply:
No, it is only applicable for elements inside the chart